### PR TITLE
add more info on `delete_from_org`

### PR DESCRIPTION
**Pull request by robynnason**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

**Review by nwangtw — APPROVED** [review]

**Comment by robynnason** [comment]
@markzegarelli @nwangtw made updates based on internal discussion on the appropriate language and clarity

File: docs/analytics/apis/user-privacy-api.md [hunk 1]
```diff
@@ -36,6 +37,8 @@ Keep these considerations in mind when using the User Privacy API.
 - After the three (3) day period, the request's status changes to `submitted`. You can't stop the job at this point.
  The deletion process removes all data associated with the user from all Amplitude's systems, including associated recovery and back-up systems.
  After the job completes, its status changes to `done`.
+- By default, deletion requests create jobs for a particular project within your organization. API calls identify this project with the api key you provide in the request. To delete a user across your entire organization, set both the `delete_from_org` and `ignore_invalid_ids` parameters to `true` in your request. The API creates one deletion job per project that contains information on the requested users.
```
Comment (nwangtw) [inline docs/analytics/apis/user-privacy-api.md]: "set both the `delete_from_org` and `ignore_invalid_ids` parameters to `true` in your request."

I think "and `ignore_invalid_ids` parameters "  is not needed

File: docs/analytics/apis/user-privacy-api.md [hunk 2]
```diff
@@ -61,11 +64,13 @@ The body parameter is required. It's the deletion request object listing the `us
...
+
+Amplitude recommends setting this to `true` if setting the `delete_from_org` parameter to `true`                                                                                                                                                                                                                    |
+| `delete_from_org`                  | Boolean. Defaults to `false`. When `false`, the request will only delete the requested users from the project identified by the given API key. When set to `true`, the requested users are deleted across your entire organization. When `true`, the request will insert the end users into a job for each project that contains data for the requested users. When `true`, the `ignore_invalid_ids` parameter is automatically set to `true`, ignoring the input.                                                                                                                        |
```
Comment (nwangtw) [inline docs/analytics/apis/user-privacy-api.md:70]: there are 3 sentences start with "When `true`" or `When set to `true`". I am wondering if we can rephrase it a little.

Comment (robynnason) [reply docs/analytics/apis/user-privacy-api.md]: reworded a bit, lmk what you think

File: docs/analytics/apis/user-privacy-api.md [hunk 3]
```diff
@@ -61,11 +64,13 @@ The body parameter is required. It's the deletion request object listing the `us
...
+| `ignore_invalid_id`                | Boolean. Defaults to `false`. With `false`, if any users in the request aren't found in the project, the API returns a 400 error, and it doesn't mark any users in the request for deletion. With `true`, the response is a 200 success that includes a list of `invalid_ids`, and the API adds any users found in the relevant project to the relevant job. `invalid_ids` represent any requested users with no data found in the relevant project. If the parameter `delete_from_org` is `true`, this field is automatically set to `true`, ignoring the input. 
+
+Amplitude recommends setting this to `true` if setting the `delete_from_org` parameter to `true`                                                                                                                                                                                                                    |
```
Comment (nwangtw) [inline docs/analytics/apis/user-privacy-api.md:69]: This should be removed?

File: docs/analytics/apis/user-privacy-api.md [hunk 4]
```diff
@@ -249,13 +248,243 @@ The body parameter is required. It's the deletion request object listing the `us
...
+    }
+    ```
+
+### Example request: Delete users from all projects in your organization
+
+=== "cURL"
+
+    ```bash
+    curl --location --request POST 'https://amplitude.com/api/2/deletions/users' \
+    -U 'API_KEY:SECRET_KEY' \
```
Comment (nwangtw) [inline docs/analytics/apis/user-privacy-api.md:292]: lower case

**Pull request merged**